lib/ansible/modules/cloud/amazon/rds_instance_facts.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
```

Comment:
I see these other modules are in a separate PR, can you remove them from this one and have the review for these facts ones over there?